nservicebus/upgrades/7to8/index.md

```diff
@@ -177,4 +177,4 @@ endpointConfiguration.EnableOutbox();
 
 ### AbortReceiveOperation
 
-`ITransportReceiveContext.AbortReceiveOperation` operation is no longer supported
+`ITransportReceiveContext.AbortReceiveOperation` has been deprecated in favour of throwing a [`OperationCancelledException`](https://docs.microsoft.com/en-us/dotnet/api/system.operationcanceledexception)
```

Comment:
```suggestion
`ITransportReceiveContext.AbortReceiveOperation` has been deprecated in favor of throwing an [`OperationCancelledException`](https://docs.microsoft.com/en-us/dotnet/api/system.operationcanceledexception).

Throwing `OperationCancelledException` will still result in recoverability policies being executed, eventually resulting in the message being sent to the error queue if the message is retried enough times, which is safer than allowing a message to get stuck in an infinite retry loop. This can be overridden using a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md) if necessary.
```